src/home/components/Form/index.tsx

```diff
@@ -26,9 +26,14 @@ const Form = ({ errors, handleSubmit, loading }: FormProps) => {
 
   return (
     <Box flexGrow={1}>
-      <form onSubmit={handleFormSubmit}>
+      <form action="post" onSubmit={handleFormSubmit}>
```

Comment:
Esse Post viria para auxiliar a a11y mano? Realmente não sei 😅

Reply:
Então, sim aparentemente não se pode usar um form sem action, dai botei mesmo q seja simbólico saka?

Reply:
to ligado q tem o method pra isso, mas fiquei na duvida doq por.

Reply:
Se tiver alguma ideia pode mandar q troco.

Reply:
De fato tava por fora, hoje tenho o costume de evitar usar a tag form, sei que por semântica quebra o html, mas é pra evitar uns efeitos colaterais que dava nos eventos e tal